src/common/core/core.js

```diff
@@ -3911,7 +3942,406 @@ define([
+         * @param {string} newName - the new name of the registry.
+         *
+         * @throws {CoreIllegalArgumentError} If some of the parameters doesn't match the input criteria.
+         * @throws {CoreIllegalOperationError} If the context of the operation is not allowed.
+         * @throws {CoreAssertError} If some internal error took place inside the core layers.
+         */
+        this.renameRegistry = function (node, oldName, newName) {
+            ensureNode(node, 'node');
+            ensureType(oldName, 'oldName', 'string');
+            ensureType(newName, 'newName', 'string');
+
+            return core.renameRegistry(node, oldName, newName);
+        };
+
+        /**
+         * Renames the given set of the node.
+         * @param {module:Core~Node} node - the node in question.
+         * @param {string} oldName - the current name of the set in question.
+         * @param {string} newName - the new name of the set.
+         *
+         * @throws {CoreIllegalArgumentError} If some of the parameters doesn't match the input criteria.
+         * @throws {CoreIllegalOperationError} If the context of the operation is not allowed.
+         * @throws {CoreAssertError} If some internal error took place inside the core layers.
+         */
+        this.renameSet = function (node, oldName, newName) {
+            ensureNode(node, 'node');
+            ensureType(oldName, 'oldName', 'string');
+            ensureType(newName, 'newName', 'string');
+
+            if (core.getOwnSetNames(node).indexOf(oldName) === -1) {
+                throw new CoreIllegalOperationError('Cannot rename nonexistent set [' + oldName + ']');
+            }
+            return core.renameSet(node, oldName, newName);
+        };
+
+        /**
+         * Returns the meta concept that defines the given attribute.
+         * @param {module:Core~Node} node - the node in question.
+         * @param {string} name - the name of the attribute in question.
+         *
+         * @return {module:Core~Node} The meta concept that defines the attribute for the given node.
+         *
+         * @throws {CoreIllegalArgumentError} If some of the parameters doesn't match the input criteria.
+         * @throws {CoreIllegalOperationError} If the context of the operation is not allowed.
+         * @throws {CoreAssertError} If some internal error took place inside the core layers.
+         */
+        this.getAttributeDefinitionOwner = function (node, name) {
+            ensureNode(node, 'node');
+            ensureType(name, 'name', 'string');
+
+            if ((core.getValidAttributeNames(node) || []).indexOf(name) === -1) {
+                throw new CoreIllegalOperationError('Not a valid attribute name [' + name + '] of the node.');
+            }
+
+            return core.getAttributeDefinitionOwner(node, name);
+        };
+
+        /**
+         * Returns the meta concept that defines the given pointer relationship.
```

Comment:
Returns the owner and the target where the pointer definition for the pair in question is defined.